app/build.gradle

```diff
@@ -44,6 +44,9 @@ android {
             buildConfigField("String", "intercomAppId", "\"\"")
             buildConfigField("String", "amplitudeKey", "\"\"")
         }
+        buildConfigField("boolean", "CRASHLYTICS_ENABLED", "false")
```

Comment:
And btw, maybe we should leave these fields in build types? Since we are going switch on/off crashlytics for feature builds. 

And maybe we'll do some actions with analytics

Reply:
I think we should keep default items in default config
and switch them to not default on needed build type

Reply:
but we can duplicate it in every buildType :) as we decide I think ))

Reply:
yes yes.. but we need to add this field in all buildTypes

so we can make 3 points where we will assign fields

for every buildType

or make it default and enable or disable in needed

so in this current case according to Firebase we can make it enabled by default and disable it on debug buildType :)

or we will need to specify it in every buildType

Reply:
im okay to put buildConfig values in the defaultConfig with default values and override them in the buildTypes if needed

Reply:
less lines in build config 😉